### ISSUE TITLE
Grayed out playlist items, "This content is not available"

### ISSUE THREAD

**Issue by busybox11** [spicetify/spicetify-cli#2106]
### 🔍 Have you checked Spicetify.app page for your issue?

- [X] I have checked the FAQ

### 🔍 Is there already an issue for your problem?

- [x] I have checked older issues, open and closed

### ℹ Environment / Computer Info

```Markdown
- Spotify version: Spotify version 1.1.84.716.gc5f8b819,
- Spicetify version: 2.16.0
```

### 📝 Description

All the songs are grayed out, and when trying to play, I have a toast saying "This content is not available".
Playback works fine (like controlling from a chrome window, or queueing via right click).
Also, MPRIS information doesn't update.

### 📸 Screenshots

![image](https://user-images.githubusercontent.com/29630035/212573428-1874aad5-204c-45cf-9e3a-36cc26db1061.png)

**Comment by busybox11** [comment]
Upon further debugging, I noticed this request is failing, and is the only one.
![image](https://user-images.githubusercontent.com/29630035/212573672-5071ec50-9c33-4fbe-8ea8-59f3b7ead22e.png)

**Comment by busybox11** [comment]
Hm. Disabling spicetify doesn't make it work again. I'm going to uninstall and reinstall, and see what happens.

**Comment by kyrie25** [comment]
Not a CLI issue.

**Comment by busybox11** [comment]
Happened after using the CLI. What should I do?

**Comment by kyrie25** [comment]
Everything on the Spotify client is restored to its original state after `spicetify restore`, thus this has something to do with the client itself.

**Comment by busybox11** [comment]
That's what I assumed too. Tried removing every single Spotify-related file on my machine, uninstalling the client, removing config files, reinstalling the client with all the dependencies without using spicetify again. I used the `spicetify-cli` AUR package. Is there any dependency installation that I'm not aware of? It maybe could be related to something like this, though I'm really confused.

**Comment by kyrie25** [comment]
This is already outside our scope so I can't provide any support whatsoever.

Though I would suspect it has to be on the network side, try looking into the error code of the request and why it failed instead, perhaps it could be a rate limit.

**Comment by busybox11** [comment]
That's the thing, there's no error code, no return thing, nothing. Only bit of somewhat helpful info I have is that it happened after applying spicetify, hence why I'm coming here.
This doesn't look like something that's on your end indeed, but I tried anyway, in case you would be familiar with similar issues.
Thanks for your help anyway, sorry if I bothered you! This might be a spotify-related problem, as it tends to happen more and more frequently lately lol

**Comment by busybox11** [comment]
Ah well, it just started working again. So this does look like a Spotify issue that happened with a very bad timing lol
I applied Spicetify again just to see and it does work. I apologize for the inconvenience again, this was really bad timing...

I still have an issue, experimental features is blank. But I didn't do any research about this for now. If I can't find a way to fix it, I'll open another issue.

Thanks for the support!

**Comment by kyrie25** [comment]
> I still have an issue, experimental features is blank. But I didn't do any research about this for now. If I can't find a way to fix it, I'll open another issue.

Experimental Features is only available on Spotify `1.1.92` and later. (#1965)

**Comment by busybox11** [comment]
Oh, makes sense. Hope Spotify will update on Linux someday then.
Thanks a lot for the heads up:
